digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -1871,13 +1876,56 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
     transformation(values)
   }
 
+  get("/userConfiguration/elys") {
+    val user = userProvider.getCurrentUser()
+    val elysId = municipalityDao.getElysByMunicipalities(user.configuration.authorizedMunicipalities)
+    val elysIdAndName = municipalityDao.getElysIdAndNamesByCode(elysId.toSet)
```

Comment:
already exist a method which return MunicipalityInfo(id, ely, name) 

> getMunicipalitiesNameAndIdByCode

Reply:
In this case I need the ely info (id and name) not the municipality info

Reply:
After talking with the team the development is correct